### PR TITLE
[VEN-2617] Multisig VIP to accept VTreasury ownership for X Layer testnet

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

**Review by chechu — COMMENTED** [review]
I would add some public RPC's to the .env.example file

**Comment by GitGuru7** [comment]
> 88cf2da

File: .env.example [hunk 1]
```diff
@@ -9,3 +9,5 @@ ARCHIVE_NODE_opbnbtestnet=https://opbnb-testnet.nodereal.io/v1/<YOUR_KEY_HERE>
 ARCHIVE_NODE_opbnbmainnet=https://opbnb-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 ARCHIVE_NODE_arbitrumsepolia=https://sepolia-rollup.arbitrum.io/rpc
 ARCHIVE_NODE_arbitrumone=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/arbitrum-nitro/
+ARCHIVE_NODE_xlayertestnet=https://rpc.ankr.com/xlayer_testnet/<YOUR_KEY_HERE>
+ARCHIVE_NODE_xlayermainnet=https://rpc.ankr.com/xlayer<YOUR_KEY_HERE>
```
Comment (chechu) [inline .env.example:13]: Some open alternatives:

- Mainnet: https://rpc.xlayer.tech/
- Testnet: https://testrpc.xlayer.tech/

**Review by chechu — APPROVED** [review]

**Comment by GitGuru7** [comment]
Multisig tx:
https://www.okx.com/web3/explorer/xlayer-test/tx/0x94cf04360d5c43a8d3237aa50f17100a89c8e17ea57a674446f5b02856d3283b

**Pull request merged**